soc/arc/snps_arc_hsdk/Kconfig.defconfig

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2019 Synopsys, Inc. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_ARC_HSDK
+
+config SOC
+	string
+	default "snps_arc_hsdk"
+
+config CPU_HS38_LINUX
+	def_bool y
+
+config NUM_IRQ_PRIO_LEVELS
+	# This processor supports 4 priority levels:
+	# 0 for Fast Interrupts (FIRQs) and 1-3 for Regular Interrupts (IRQs).
+	default 2
```

Comment:
processor supports 4 levels, but Zephyr only 2? Please add a note on this if so, or other rationale for configuring 2 instead of 4.

Reply:
the comment is error, hsdk just supports 2 levels, i made a fix